src/types/chainTypes.ts

```diff
@@ -33,8 +33,8 @@ export interface RoundInfo extends Struct {
 }
 
 export interface Delegator extends Struct {
-  delegations: Vec<Stake>
-  total: Balance
+  owner: AccountId
+  amount: Balance
 }
```

Comment:
probably the whole file can go now that we have api augmentation, but we'll save that for another PR

Reply:
yes, absolutely. I could also do that